### DISCUSSION TITLE
「开源之史」系列之十二/四：GPLv3 —— 软件自由的分裂还是捍卫？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#71] [Announcements]
# 「开源之史」系列之十二/四：GPLv3 —— 软件自由的分裂还是捍卫？ - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-08-freedom-forever-gplv3/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实, 从  GPL 协议不同版本撰写过程对比可感受到:

- 当创作只面对本心时, 一切是清晰明快的, 可以一挥而就
- 但是, 当作品进入社会, 并和越来越多事务产生关联时, 羁绊就产生了
- 而且, 随着关注的人越来越多, 研究的人越来越多, 当初的任何一丝天真都将变成漏洞而被利用
- 可惜, 想补上漏洞, 就不象宪法修正案那么可预期了, 如果没有一个全社会都认同的流程, 无论在自营网站中进行什么规模的意见收集整理, 都将导致两个结果:

0: 文本法务化, 常人不可读
1: 引发更多不满

这也是 `欲戴其冠, 必承其重`

GPLv3 也不可能是终点, 元宇宙到来了, 在 meta 中 GPL 应该保护什么, 又如何保护, 想来已经开始思考了吧?

**Reply by lijiangsheng1** [reply]
Technology is always at least one step ahead of the copyright law.

技术不变，就没有许可，技术在变化，许可就需要对应的变化。
